src/components/Filters/Filters.tsx

```diff
@@ -255,6 +262,7 @@ const Filters: React.FC<FiltersProps> = ({
       })}
       <BottomBar
         hasUnappliedFilters={hasUnappliedFilters}
+        isApplyDisabled={hasInvalidValues}
```

Comment:
Be aware that Filters are using form validation with submit button inside `ButtonBar`. Then there is mismatch of all validations together, your custom and native. We want to get rid of the native validation message. Pls see the screenshot 
<img width="452" alt="Screen Shot 2021-04-21 at 11 21 52" src="https://user-images.githubusercontent.com/6930850/115530204-d9a08500-a293-11eb-8bf8-47bb801920a0.png">